commands/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Sample Commands Extension",
+  "description": "Press Ctrl+Shift+Y to send an event (Command+Shift+Y on a Mac).",
+  "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/commands",
+  "manifest_version": 2,
+  "version": "1.0",
+  "background": {
+    "scripts": ["background.js"]
+  },
+  "applications": {
+    "gecko": {
+      "id": "commands@mozilla.org"
+    }
+  },
+  "commands": {
+    "toggle-feature": {
+      "suggested_key": { "default": "Ctrl+Shift+Y" },
```

Comment:
I just tested Chrome's sample extension that uses "Ctrl+Shift+Y" and noticed that it doesn't work on Chrome either - it did work when I switched to "Ctrl+Shift+5".   Do you know why that is? I chose the shortcut by referencing the API page, which says that the supported keys are: A-Z, 0-9, Comma, Period, Home, End, PageUp, PageDown, Space, Insert, Delete, Arrow keys (Up, Down, Left, Right) and the Media Keys (MediaNextTrack, MediaPlayPause, MediaPrevTrack, MediaStop).